source/__tests__/integration/create-property.test.js

```diff
@@ -215,17 +217,16 @@ describe('property URI and Label are required', () => {
       propertyURI: 'not-a-uri',
       propertyLabel: 'propLabel'
     })
-    // wait for resourceURI check
+    // wait for resourceURI and propertyURI checks
+    await page.waitFor(1000, {waitUntil: 'networkidle2'})
```

Comment:
I'm pretty confident this pause helps

Reply:
neat, this is a nice thing to be able to wait for.  like what i might do as a human tester when unsure whether a page was done loading (i had to read up a bit on what `networkidle` is).